test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -15,6 +17,29 @@ export default class PossibleIssues extends Component {
         await this.fetchIssues();
     }
 
+    getUserFeedback = async(repoName, buildName, issueName, issueCreator, accuracy) => {
+        if (accuracy === true) {
```

Comment:
Whether `accuracy` is true or false, we need to pass to backend, along with this `accuracy` value.